lib/state.js

```diff
@@ -1092,6 +1092,7 @@ class State {
       'SELECT region, az, "instanceType", "workerType", code, called AS time, message',
       'FROM awsrequests',
       'WHERE error = true AND method = \'runInstances\'',
+      'AND code <> \'UnauthorizedOperation\'',
```

Comment:
I'm guessing that this is here to avoid leaking information about our iam policies? It might be a bit confusing that some errors "arbitrarily" will not show up in the UI. Is there another way we can handle this. It might be fine to just show them as well.